cli/script/command-parser.ts

```diff
@@ -316,7 +316,7 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
         isValidCommandCategory = true;
         isValidCommand = true;
         yargs.usage(USAGE_PREFIX + " login [options]")
-            .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
+            .demand(/*count*/ 0, /*max*/ 0)  
```

Comment:
I think we still want max to be one here. We have an undocumented argument that can be used for testing, eg:

`code-push login http://localhost:3000`